### PR TITLE
Improve getTimes from DxpClass

### PR DESCRIPTION
**Pull request by gabrielfedel**
Fix the getCountTime and added getRealTime, where realTime = LiveTime + DeadTime

### CODE REVIEW

**Comment by hhslepicka** [comment]
@gabrielfedel How is this DXP being used at the beam line?
Why do you need to specify the acquire time per channel?
Isn't it being used with the MCA?

File: py4syn/epics/DxpClass.py [hunk 1]
```diff
@@ -104,7 +108,19 @@ def setCountTime(self, time):
         self.timer = Timer(time + self.responseTimeout)
 
     def getCountTime(self):
-        return self.pvDxpTime.get()
+        # AcquireTimes are the same
+        return self.pvDxpAcquireTime[0].get()
```
Comment (hhslepicka) [inline py4syn/epics/DxpClass.py:112]: Please define a constant for ```0``` to make the code more clear.

File: py4syn/epics/DxpClass.py [hunk 2]
```diff
@@ -104,7 +108,19 @@ def setCountTime(self, time):
...
+
+    def getRealTime(self):
+        """Return the Real Time"""
+        if self.channels == 1:
+            return self.pvDxpRealTime[1].get()
```
Comment (hhslepicka) [inline py4syn/epics/DxpClass.py]: Please define a constant for ```1``` to make the code more clear.

**Review by hhslepicka — CHANGES_REQUESTED** [review]

**Comment by hhslepicka** [comment]
Hi @gabrielfedel it is still unclear why you need a set of real and acquire time per channel.
Would you please clarify? I believe that this device is a bit messy.
I can understand that it probably works at the beam line but the use seems very weird to me.

**Comment by gabrielfedel** [comment]
Hi @hhslepicka , this device could control more than one detector, and it is possible to set different acquisition time for each. 
Do you think I can improve the code? Or, maybe explain better?

**Comment by hhslepicka** [comment]
@gabrielfedel, for now we keep it like that and we can talk offline just so I can understand better the idea behind this device and help you with that.
Merging as is.

**Review by hhslepicka — APPROVED** [review]

**Pull request merged**